### PR TITLE
Feature: Configurable vertical align in <Overlay /> component

### PR DESCRIPTION
**Pull request by hzub**

### CODE REVIEW

File: packages/overlay/src/index.tsx [hunk 1]
```diff
@@ -251,6 +263,16 @@ export const Overlay: React.FC<OverlayProps> = ({
   )
 }
 
+type OverlayVerticalAlign =
```
Comment (Idered) [inline packages/overlay/src/index.tsx]: This should contain only `top, center, bottom` or `start, center, end`, we don't need all these values. Right?

Comment (hzub) [reply packages/overlay/src/index.tsx]: Correct, left `stretch` anyway due to different behavior than `top`

File: packages/overlay/src/index.tsx [hunk 2]
```diff
@@ -57,6 +72,7 @@ export const Overlay: React.FC<OverlayProps> = ({
   shouldCloseOnClick = true,
   shouldCloseOnEscapePress = true,
   preventBodyScrolling = false,
+  verticalAlign = 'stretch',
```
Comment (Idered) [inline packages/overlay/src/index.tsx]: Current default is `center`, isn't it?

Comment (hzub) [reply packages/overlay/src/index.tsx]: Current resulting behavior is `stretch`, which basically was an incentive for creating this PR :D

**Pull request merged**